visidata/loaders/toml.py

```diff
@@ -0,0 +1,51 @@
+from visidata import (
+    ColumnItem,
+    PythonSheet,
+    VisiData,
+    asyncthread,
+    deduceType,
+    vd,
+)
+
+
+@VisiData.api
+def open_toml(vd, p):
+    return TomlSheet(p.name, source=p)
+
+
+class TomlSheet(PythonSheet):
+    """A Sheet representing the top level of a loaded TOML file.
+
+    This is an intentionally minimal loader with cues taken from
+    VisiData built-in JSON and Python object sheet types.
+    """
```

Comment:
@ajkerrigan This is such a thoughtfully put together PR.

I think the only thing that concretely felt like it might be useful to add is some `help` text for the sidebar. I am thinking something like `zEnter` to dive into the Dictionary structures. I am going to put a prototype together!

Reply:
I added a help string!

@ajkerrigan please review! I found it tricky to try to describe `z Enter` and `expand-col` in non-jargon language.